src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -449,40 +577,63 @@ void CoilCoolingDX::getDataAfterSizing(Real64 &_normalModeRatedEvapAirFlowRate,
     _normalModeRatedCapacity = this->performance.normalMode.ratedGrossTotalCap;
 }
 
+CoilCoolingDXCurveFitSpeed & CoilCoolingDX::normModeNomSpeed()
+{
+    return this->performance.normalMode.speeds[this->performance.normalMode.nominalSpeedIndex];
+}
```

Comment:
Ah, these are functions. This syntax is just confusing to me. See what you learn reviewing new code.

This:
this->normModeNomSpeed().condenser_air_flow_rate;

is the same as:
this->performance.normalMode.speeds[this->performance.normalMode.nominalSpeedIndex].condenser_air_flow_rate;